force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +243,15 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
+        List<Integer> integerList = new List<Integer>();
 
         // Add the integers 1, 2, and 3 to the list
-        // add 1
-        // add 2
-        // add 3
+        integerList.add(1); // add 1
+        integerList.add(2); // add 2
+        integerList.add(3); // add 3
```

Comment:
Try to instantiate the list in the constructor
https://salesforce.stackexchange.com/questions/203694/how-to-initialize-list-set-values-by-construction-in-apex